control-plane-operator/controllers/hostedcontrolplane/clusterpolicy/deployment.go

```diff
@@ -70,6 +70,12 @@ func buildOCMContainerMain(image string) func(*corev1.Container) {
 			"--namespace=openshift-kube-controller-manager",
 		}
 		c.VolumeMounts = volumeMounts.ContainerMounts(c.Name)
+		c.Env = []corev1.EnvVar{
```

Comment:
adding this env var to allow policy-controller create events in `openshift-kube-controller-manager` namespace

Reply:
can you add this comment as a comment in code?